Source/SubtitleParser.cpp

```diff
@@ -0,0 +1,59 @@
+#include "pch.h"
+#include "SubtitleParser.h"
+#include "SubtitleParser.g.cpp"
+#include "FFmpegMediaSource.h"
+#include "MediaSourceConfig.h"
+
+namespace winrt::FFmpegInteropX::implementation
+{
+    winrt::Windows::Foundation::IAsyncOperation<winrt::FFmpegInteropX::SubtitleParser> SubtitleParser::ReadSubtitleAsync(winrt::Windows::Storage::Streams::IRandomAccessStream stream,
+        hstring streamName,
+        winrt::FFmpegInteropX::MediaSourceConfig config,
+        winrt::Windows::Media::Core::VideoStreamDescriptor videoDescriptor,
+        uint64_t windowId,
```

Comment:
Despite of not being used yet - what would be the purpose of supplying a  windowId?

Isn't it only used to automatically determine whether HDR is enabled on the app's display? Why would it be required for subtitles then?

Reply:
I guess windowId and hseHdr are copy+paste error. The VideoStreamDescriptor makes sense, should be used to initialize the SubtitleStreamProvider with the video size. It's important for image based subtitle streams, to calculate the best target position.

Reply:
Fixed.